FriendlySkeletonWand/Items/NecromancerHood.cs

```diff
@@ -14,35 +14,54 @@ internal class NecromancerHood : Item
 
         public static ConfigEntry<int> necromancySkillBonus;
 
+        public static ConfigEntry<CraftingTable> craftingStationRequired;
+        public static ConfigEntry<int> craftingStationLevel;
+        public static ConfigEntry<string> craftingCost;
+
         public override void CreateConfigs(BaseUnityPlugin plugin)
         {
             base.CreateConfigs(plugin);
 
-            allowed = plugin.Config.Bind("Server config", "NecromancerHoodAllowed",
+            allowed = plugin.Config.Bind("NecromancerHood (Server Synced)", "NecromancerHoodAllowed",
                 true, new ConfigDescription("Whether crafting a Necromancer's Hood is allowed or not.", null,
                 new ConfigurationManagerAttributes { IsAdminOnly = true }));
 
-            necromancySkillBonus = plugin.Config.Bind("Server config", "NecromancerHoodSkillBonus",
-                10, new ConfigDescription("How much wearing the item should raise the Necromancy level (set to 0 to have no set effect at all).", null,
+            craftingStationRequired = plugin.Config.Bind("NecromancerHood (Server Synced)", "Necromancer Hood Crafting Station",
```

Comment:
I think the spaces in the 2nd part should be removed. I don't think they cause any problems, but it just breaks with the convention I've seen established in other mods. So `NecromancerHoodCraftingStation` would be better even if it is a little more unreadable. It's also the way all the other configs in this mod are set up.

Granted, I have not gone and surveyed dozens of mods and how they do their configs. But the ones I've seen have all been like that.